Domain/Aggregates/Products/ValueObjects/Money.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Domain.Common;
+
+namespace Domain.Aggregates.Products.ValueObjects
+{
+    public class Money : ValueObject
+    {
+        public decimal Value { get; }
+        public string Currency { get; }
```

Comment:
i think it's better to use init keyword as this value object data should not have any accessible to modify after initialization .